src/web/mjs/connectors/Futekiya.mjs

```diff
@@ -0,0 +1,73 @@
+import SpeedBinb from './templates/SpeedBinb.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Futekiya extends SpeedBinb {
+
+    constructor() {
+        super();
+        super.id = 'futekiya';
+        super.label = 'futekiya';
+        this.tags = ['manga', 'english'];
+        this.url = 'https://read.futekiya.com';
+    }
+
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, '.card-body.book-detail h3');
+        let id = uri.pathname;
+        let title = data[0].innerText.trim();
+        return new Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        let maxpagecount = 1;
+        let manga = [];
+        for (let i = 1; i <= maxpagecount; i++) {
+            const request = new Request(new URL('/browse?page=' + i, this.url), this.requestOptions);
+            if (maxpagecount == 1) {
+                const newpagecount = await this.fetchDOM(request, ".pagination :nth-last-child(2)");
+                maxpagecount = parseInt(newpagecount[0].innerText.trim());
```

Comment:
Can this be done before starting the loop?
This would avoid the conditional check.

Reply:
good catch, my intention first was to only use fetchDOM once but as I did use two in the end adding it to the for loop doesn't help anyone.